src/runtime-rs/crates/hypervisor/src/ch/inner_device.rs

```diff
@@ -397,7 +401,7 @@ mod tests {
 
         let v: [u8; 6] = [10, 11, 128, 3, 4, 5];
         let mac_address = Address(v);
-        cfg.guest_mac = Some(mac_address.clone());
+        cfg.guest_mac = Some(mac_address);
```

Comment:
@Apokleos how this change relates to the PR ?

Reply:
Doing `cargo fmt && clippy` with a low rust version causes the problem.
fixed.

Reply:
> Doing `cargo fmt && clippy` with a low rust version causes the problem. fixed.

Not sure to understand what the problem is but in any case this change is certainly not related to this commit. If it is really needed for this PR to build, then put this change in its own commit with a clear explanation. Otherwise just drop it. We want our commits to be focused and precise for the sake of long term maintainability.

Reply:
Fixed
Thx for your suggestion